malboxes/templates/win7_64_analyst.json

```diff
@@ -31,10 +31,14 @@
 	{% if tools_path %},
 		{% include 'snippets/tools.json' %}
 	{% endif %}
-        {% if ida_path %},
```

Comment:
These lines got removed to fix the space/tab indent mixing

Reply:
Yes, I noticed that. Thanks! My vim config is still badly configured for javascript/json, at some point I'll rewrite everything into 4 spaces but not now.